easybuild/easyconfigs/o/OpenSees/OpenSees-3.2.0-intel-2020a-Python-3.8.2-parallel.eb

```diff
@@ -23,7 +23,7 @@ patches = [
 checksums = [
     '200f70fc86b7c4c2978eaef1aacd206cb40791fa797e1b9cc51da9633b53eb36',  # v3.2.0.tar.gz
     # OpenSees-3.2.0-add_Makefile_def_parallel.patch
-    'a5af4dc405916575a3eaea8cc78fb17c9e016cacd7f1048f906433ecd4e9da5a',
+    '96e541d9827cabe0cef64e42c6d8e6581837bc6dc4a47b476a2b76a9636d139e',
```

Comment:
@mckee107 It looks like `d9eca142b325b0c64d9e1db11f2eb994af86530c1262ed29eec44dc2d75b3ea1` is the correct SHA256 checksum after changing the patch file?

Note that you can use `eb OpenSees-3.2.0-intel-2020a-Python-3.8.2-parallel.eb  --inject-checksums --force` to correct this automatically.

Reply:
I am getting `96e541d9827cabe0cef64e42c6d8e6581837bc6dc4a47b476a2b76a9636d139e` on my computer, but I may be getting it incorrectly. I can update the easyconfig to use `d9eca142b325b0c64d9e1db11f2eb994af86530c1262ed29eec44dc2d75b3ea1` if you would like.

Reply:
How exactly are you determining the checksum locally?

Reply:
Using git bash. Tried both my local repo and the pr fork from easybuild/easyconfigs. If I'm doing it wrong, please let me know.

```
codim@DESKTOP-BVSQJN9 MINGW64 ~/Documents/GitHub/easybuild-easyconfigs/easybuild/easyconfigs/o/OpenSees (pr/12403)
$ shasum -a 256 OpenSees-3.2.0-add_Makefile_def_parallel.patch
96e541d9827cabe0cef64e42c6d8e6581837bc6dc4a47b476a2b76a9636d139e *OpenSees-3.2.0-add_Makefile_def_parallel.patch
```

Reply:
That doesn't agree with what I'm seeing when I have your `develop` branch checked out:

```
$ sha256sum easybuild/easyconfigs/o/OpenSees/OpenSees-3.2.0-add_Makefile_def_parallel.patch
d9eca142b325b0c64d9e1db11f2eb994af86530c1262ed29eec44dc2d75b3ea1  easybuild/easyconfigs/o/OpenSees/OpenSees-3.2.0-add_Makefile_def_parallel.patch

$ shasum -a 256 easybuild/easyconfigs/o/OpenSees/OpenSees-3.2.0-add_Makefile_def_parallel.patch
d9eca142b325b0c64d9e1db11f2eb994af86530c1262ed29eec44dc2d75b3ea1  easybuild/easyconfigs/o/OpenSees/OpenSees-3.2.0-add_Makefile_def_parallel.patch
```

What kind of environment are you computing the SHA256 checksum? I see MINGW64, so is this Cygwin? That could explain it, maybe...

Reply:
I was using it on windows, MINGW64.  I tried again on the cluster login node, and got a value that agrees with you.
```
[mckee107@ada4 easyconfigs]$ shasum -a 256 o/OpenSees/OpenSees-3.2.0-add_Makefile_def_parallel.patch
d9eca142b325b0c64d9e1db11f2eb994af86530c1262ed29eec44dc2d75b3ea1  o/OpenSees/OpenSees-3.2.0-add_Makefile_def_parallel.patch
```

I'll update the file